packages/mobile/src/screens/user-list-screen/UserChip.tsx

```diff
@@ -51,9 +53,9 @@ export const UserChip = (props: UserChipProps) => {
       <Pressable style={styles.details} onPress={handlePress}>
         <ProfilePhoto profile={user} style={styles.photo} />
         <View>
-          <Text style={styles.handle}>{handle}</Text>
+          <UserBadges user={user} badgeSize={10} nameStyle={styles.name} />
```

Comment:
I've been wanting to move the name out of `UserBadges` entirely

Reply:
ahh great to hear, i was also thinking it would be good to remove the name, will update